scripts/generate_samples.ts

```diff
@@ -0,0 +1,88 @@
+import fs from "fs";
+import { chromium } from "@playwright/test";
+
+// Declare a global augmentation for the Window interface
+declare global {
+  interface Window {
+    map: {
```

Comment:
```suggestion
    map?: {
```

Based on usage below this could be undefined.